CHANGELOG.md

```diff
@@ -13,21 +13,26 @@ Try to use the following format:
 ### Changed
 ### Fixed
 
+## [13.17.2]
+
+### Changed
+- Only fastq files older than three weeks will be compressed
+
+
 ## [13.7.1]
```

Comment:
can you please fix this typo? :stuck_out_tongue:

Reply:
Which one?

Reply:
Changed to `13.17.1`